### ISSUE TITLE
Do you think of adding child nodes in the left of root child?

### ISSUE THREAD

**Issue by iamhiutrun** [g-zato/Mind-Mapping-App#1]
I'm confused about it, it's nice for me if you suggest me a solution?

**Comment by guchengxi1994** [comment]
I think that maybe impossible, the [graphview](https://github.com/nabil6391/graphview) dependence does not support.

**Comment by g-zato** [comment]
TLDR: Not at the moment.

Hi all, I'm so sorry about the late response. I honestly have no idea how to do that, since I'm using only graphview's core features, and it draws tree-like structures in only one direction. We could try simply mirroring a second tree, but then I'd have no idea how how to id the nodes XD.

This project was developed originally to build chatbot dialog flows, that also explains how the app's functionality was planned out. I haven't touched on it since last year, because of other stuff ( sorry :( ).

Do you mind telling me how you're using the project? I honestly didn't think anyone would be interested in this prototype hahaha
